app/controllers/container_controller.rb

```diff
@@ -111,6 +111,7 @@ def identify_container(id = nil)
 
   # ST clicked on in the explorer right cell
   def x_show
+    get_tagdata(Container.find_by_id(from_cid(params[:id])))
```

Comment:
@h-kataria Is this the correct place to put this line? I initially thought that that ```show``` would be inherited from the mixin. 
cc @abonas